Makefile

```diff
@@ -195,15 +169,6 @@ clean-tizen-arm:
 clean-tizen-x86:
 	rm -rf ${TIZEN_X86_BUILD_DIR}
 
-clean-ios-framework:
-	rm -rf ${IOS_FRAMEWORK_BUILD_DIR}
-
-clean-ios-framework-sim:
-	rm -rf ${IOS_FRAMEWORK_SIM_BUILD_DIR}
-
-clean-ios-framework-universal:
-	rm -rf ${IOS_FRAMEWORK_UNIVERSAL_BUILD_DIR}
-
 android: android-demo
 	@echo "run: 'adb install -r android/demo/build/outputs/apk/demo-debug.apk'"
```

Comment:
Not part of this review, but we should update the path here to be: `platforms/android/demo/build/outputs/apk/debug/demo-debug.apk`

Reply:
Guess I might as well fix that path while I'm messing around in the Makefile - will update.